pkg/reconciler/v1alpha1/route/route_test.go

```diff
@@ -143,7 +143,7 @@ func getTestRevisionForConfig(config *v1alpha1.Configuration) *v1alpha1.Revision
 func getActivatorDestinationWeight(w int) v1alpha3.DestinationWeight {
```

Comment:
rename

Reply:
Actually I completely removed it, since it seems to be unused